python-frontend/src/test/java/org/sonar/python/semantic/FullyQualifiedNameTest.java

```diff
@@ -290,6 +298,17 @@ public void two_usages_callee_symbol() {
     assertThat(callExpression.calleeSymbol().usages()).extracting(Usage::kind).containsExactly(Usage.Kind.OTHER, Usage.Kind.OTHER);
   }
 
+  @Test
+  public void import_wildcard() {
+    FileInput tree = parse(
+      new SymbolTableBuilder("", pythonFile("my_module.py")),
+      "from my_module import *",
+      "from . import *"
```

Comment:
This test was introduced to maximize coverage of the existing clause which was made wider (`Scope.java:189`). But existing preconditions were not tested because they are corner ones.

Moved it to a different file, added comments